app/assets/javascripts/discourse/lib/plugin-api.js.es6

```diff
@@ -983,6 +984,31 @@ class PluginApi {
   addGlobalNotice(id, text, options) {
     addGlobalNotice(id, text, options);
   }
+
+  /**
+   * Used for decorating the `cooked` content of a plugin-outlet after it's been rendered
```

Comment:
```suggestion
   * Used for decorating the rendered HTML content of a plugin-outlet after it's been rendered
```
Cooked means something specific to us -- taking raw text and rendering markdown -- this change makes it clear that it's the HTML in the outlet.